libr/util/num.c

```diff
@@ -7,32 +7,6 @@
 #include <r_util.h>
 #define R_NUM_USE_CALC 1
 
-R_API ut16 r_num_ntohs (ut16 foo) {
-#if LIL_ENDIAN
-	ut8 *p = (ut8*)&foo;
-	foo = p[1] | p[0]<<8;
-#endif
-	return foo;
-}
-
-#define __htonq(x) (\
```

Comment:
?

> On 28 Apr 2016, at 20:13, Damien Zammit <notifications@github.com> wrote:
> 
> In libr/util/num.c:
> 
> > @@ -7,32 +7,6 @@
> >  #include <r_util.h>
> >  #define R_NUM_USE_CALC 1
> >  
> > -R_API ut16 r_num_ntohs (ut16 foo) {
> > -#if LIL_ENDIAN
> > -	ut8 *p = (ut8*)&foo;
> > -	foo = p[1] | p[0]<<8;
> > -#endif
> > -	return foo;
> > -}
> > -
> > -#define __htonq(x) (\
> what a clusterfuck
> 
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
>